src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/preferences/PreferencesPageObject.java

```diff
@@ -34,6 +35,8 @@
   private WebElement facebookPasswordInput;
   @FindBy(css = "#facebook input[name='login']")
   private WebElement facebookSubmitButton;
+  @FindBy(css = "#mw-input-wpusenewrc")
+  private WebElement Use_advanced_recent_changes_checkbox;
```

Comment:
- naming convention is similar as below
- you should avoid using the same name for functions and for variables